doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/index.md

```diff
@@ -9,134 +9,142 @@ sort_idx: 20
 {% assign root_url = page.url | split: '/'%}
 {% capture root_url  %} /{{root_url[1]}}/{{root_url[2]}}/{% endcapture %}
 
-## Introduction
+Kaa C++ SDK is a cross-platform implementation of Kaa Endpoint.
+It enables IoT functionality on any platform which provides C++ runtime.
 
-Kaa C++ SDK is a cross-platform implementation of Kaa endpoints.
-It enables high-performance IoT functionality on any platform which provides C++ runtime.
-This means that you can deploy C++ clients on any hardware platform supported by Linux.
+## Field of application
 
-In case you need to run Kaa endpoints on more low-end hardware, consider using [Kaa C SDK]({{root_url}}/Programming-guide/Using-Kaa-endpoint-SDKs/C).
+Kaa C++ SDK is a good choice for major operating systems and high-end embedded platforms.
+It is good at building high-performance gateways and endpoints for you IoT infrastructure.
 
-This page describes build configuration for Kaa C++ SDK.
-For detailed installation instructions for different platforms, check the corresponding subpages.
+In case you need to run Kaa endpoints on bare metal hardware, consider using [the C SDK]({{root_url}}/Programming-guide/Using-Kaa-endpoint-SDKs/C).
+
+For in-depth architecture overview, refer to [C++ SDK architecture overview page]({{root_url}}/Customization-guide/Endpoint-SDKs/C++-SDK/Architecture-overview/) page.
+
+## Environment setup
+
+The recommended way to setup required dependencies is by using [Nix package mananger]({{root_url}}/Customization-guide/Endpoint-SDKs/C-SDK/Environment-setup/Nix-guide).
```

Comment:
Nix is not recommended for the end user. The user should install all dependencies using the native method for his platform. Nix is nice for developing the SDK itself, but not for building end applications.